autoload/neomake/makers/ft/go.vim

```diff
@@ -27,3 +35,15 @@ function! neomake#makers#ft#go#golint()
             \ '%-G%.%#'
         \ }
 endfunction
+
+function! neomake#makers#ft#go#gometalinter()
+    return {
+        \ 'args': ['-t', '%:p:h'],
```

Comment:
Consider passing -j here. The default setting of 16 concurrency can slam a system on larger projects.